src/testUtils.ts

```diff
@@ -340,7 +340,7 @@ export function cancelRunningTests(): Thenable<boolean> {
 function expandFilePathInOutput(output: string, cwd: string): string {
 	let lines = output.split('\n');
 	for (let i = 0; i < lines.length; i++) {
-		let matches = lines[i].match(/^\s*(.+.go):(\d+):/);
+		let matches = lines[i].match(/^\s*([(\.\/)?\w+\s*]+.go):(\d+):/);
```

Comment:
Things inside the `[]` dont need the quantifiers like `?`, `+` or `*`. `[]` defines the allowed characters not their quantity or order.

Also, this will not work for Windows.

Can you make this an exported function and add some tests to cover the different scenarios?

You can add the tests in https://github.com/Microsoft/vscode-go/blob/master/test/unit/utils.test.ts